src/pages/GamePage.jsx

```diff
@@ -29,19 +29,22 @@ const GamePage = () => {
         setUserNumber(userNumberHandler);
       }
     };
-    const checkNumber = () => {
-      if (ranArr.includes(userNumber)) {
-        console.log('traf');
-     } else {
-        console.log('nie traf')
-     }
+
+
+    const checkNumber = (arr, number) => {
+      arr.forEach((element)=>{
+        if (element === number) {
+          setResultHandler(true);
+          console.log(resultHandler);
+        }
```

Comment:
niby spoko. Na pewno zadziała, ale jeżeli masz 10000 elementów, a szukany jest na pierwszym, to i tak będzie Ci sprawdzał całą pulę. 
Masz pomysł jak to zmienić?